docs/content/doc/help/faq.en-us.md

```diff
@@ -133,8 +134,11 @@ There are multiple things you can combine to prevent spammers.
 2. Setting `ENABLE_CAPTCHA` to `true` in your `app.ini` and properly configuring `RECAPTCHA_SECRET` and `RECAPTCHA_SITEKEY`
 3. Settings `DISABLE_REGISTRATION` to `true` and creating new users via the [CLI]({{< relref "doc/usage/command-line.en-us.md" >}}), [API]({{< relref "doc/advanced/api-usage.en-us.md" >}}), or Gitea's Admin UI  
 
-### Only allow/block certain email domains
-If using OpenID, you can configure `WHITELISTED_URIS` or `BLACKLISTED_URIS` in your `app.ini`  
+### Only allow certain email domains
+You can configure `EMAIL_DOMAIN_WHITELIST` in your app.ini under `[service]`
+
+## Only allow/block certain OpenID providers
+You can configure `WHITELISTED_URIS` or `BLACKLISTED_URIS` under `[openid]` in your `app.ini`
```

Comment:
I could be wrong, but I believe the double space is needed at the end of this line so that the note goes on the next line.
```suggestion
You can configure `WHITELISTED_URIS` or `BLACKLISTED_URIS` under `[openid]` in your `app.ini`  
```

Reply:
I thought it was something I added by accident while editing :)